load/UpdateDBPFilesetTables.py

```diff
@@ -73,8 +73,27 @@ def __init__(self, config, db, dbOut, languageReader):
 		self.textUpdater = UpdateDBPTextFilesets(self.config, self.db, self.dbOut)
 		self.booksUpdater = UpdateDBPBooksTable(self.config, self.dbOut)
 		self.languageReader = languageReader
+		dbConn = SQLUtility(self.config)
+		self.updateBibleFilesSecondary = UpdateDBPBibleFilesSecondary(self.config, dbConn, self.dbOut, self.languageReader)
 
+	def processFilesSecondary(self, inputFileset):
+		""" Process secondary files for a given fileset.
+		This method is called when the fileset is already in the database, and we need to update its secondary files.
+		It checks the type of the fileset and updates the secondary files accordingly.
+		"""
+		bucket = self.config.s3_vid_bucket if inputFileset.typeCode == "video" else self.config.s3_bucket
+		lptsDBP = UpdateDBPLPTSTable(self.config, self.dbOut, self.languageReader)
 
+		if inputFileset.typeCode in {"audio", "video"}:
```

Comment:
the actions are the same for both paths. 
write a function shouldUpdateFilesSecondary and then put the logic in that function.